app/models/API1812/PenaltyDetails.scala

```diff
@@ -16,17 +16,34 @@
 
 package models.API1812
 
+import config.AppConfig
 import models.API1812.latePaymentPenalty.LatePaymentPenalty
-import play.api.libs.functional.syntax.unlift
-import play.api.libs.json.{Reads, Writes, __}
+import play.api.libs.functional.syntax.toFunctionalBuilderOps
+import play.api.libs.json.{Json, Reads, Writes, __}
+import services.DateService
 
-case class PenaltyDetails(LPPDetails: Option[Seq[LatePaymentPenalty]])
+case class PenaltyDetails(LPPDetails: Option[Seq[LatePaymentPenalty]],
+                          breathingSpace: Option[BreathingSpace]) {
```

Comment:
Breathing space is actually an array, so there could be multiple sets of BS. Unfortunately that means a bit of extra logic as you'll have to check if the user is in BS for any of the periods.